batch7crud-roll4/src/main/java/com/lftechnology/batch7crud/constant/EntityConstants.java

```diff
@@ -0,0 +1,12 @@
+package com.lftechnology.batch7crud.constant;
+
+/**
+ * Created by pratishshr on 1/21/16.
+ */
+public final class EntityConstants {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)